streaming/src/java/org/apache/hive/streaming/AbstractRecordWriter.java

```diff
@@ -393,6 +393,11 @@ public void close() throws StreamingIOFailure {
     if (LOG.isDebugEnabled()) {
       logStats("Stats after close:");
     }
+    try {
+      this.fs.close();
+    } catch (IOException e) {
+      LOG.error("Error while closing FileSystem", e);
```

Comment:
I think we should propagate this error as a RuntimeException instead of swallowing it.
If FileSystem fails to close, that would signal a serious problem.

Reply:
@EricWohlstadter Done. Thanks.